core/amber/src/main/resources/python_udf/topic_modeling.py

```diff
@@ -0,0 +1,76 @@
+import texera_udf_operator_base
```

Comment:
can we rename the file to be `topic_modeling_trainer.py`, so as the class `TopicModelingTrainer`? as it is extending from `Operator`.